silkrpc/commands/eth_api.cpp

```diff
@@ -148,11 +148,43 @@ asio::awaitable<void> EthereumRpcApi::handle_eth_get_block_by_hash(const nlohman
     co_return;
 }
 
+asio::awaitable<void> EthereumRpcApi::handle_eth_get_block_transaction_count_by_hash(const nlohmann::json& request, nlohmann::json& reply) {
```

Comment:
Add comment `// https://eth.wiki/json-rpc/API#eth_getblocktransactioncountbynumber` at line 150. Please be aware that case matters and should be exactly this one

Reply:
ok